pages/report.vue

```diff
@@ -0,0 +1,303 @@
+<template>
+  <div class="page">
+    <Card>
+      <div class="content">
+        <div>
+          <h1 class="card-title-adjustments">Submit a Report</h1>
+          <div class="">
+            <p>
+              We're committed to maintaining a safe and trustworthy platform. If you encounter
+              content that violates our
```

Comment:
Who is "we"? Seems obvious to us, I know, but this should be made less ambiguous. I would prefer the use of the first sentence from the original report modal:
> Modding should be safe for everyone, so we take abuse and malicious intent seriously at Modrinth.

Otherwise I like what you've written.